RecoTracker/TrackProducer/interface/TrackProducerBase.icc

```diff
@@ -228,9 +228,9 @@ TrackProducerBase<T>::setSecondHitPattern(Trajectory* traj, T& track,
         auto const & detWithState = it->compatibleDets(tsos,*localProp,estimator);
         if(!detWithState.size()) continue;
         DetId id = detWithState.front().first->geographicalId();
-        MeasurementDetWithData measDet = measTk->idToDet(id);
-        //if(measDet->isActive() && !measDet->hasBadComponents(detWithState.front().second)){
-        if(measDet.isActive()){ 
+        MeasurementDetWithData const & measDet = measTk->idToDet(id);
+        if(measDet.isActive() && !measDet.hasBadComponents(detWithState.front().second)){
+//        if(measDet.isActive()){ 

```

Comment:
it was already there (the opposite)
I just moved the comments from one line to the other...

not obvious, no reason to be configurable at run time.
In my opinion it was removed because of the bug (a factor 400 in the area covered by the Roc) that was giving too many false positive.
If (mini-aod) users wants count inactive as missing they can just ignore MissingInner all-together...